include/rospack/rospack.h

```diff
@@ -146,6 +146,7 @@ class ROSPACK_DECL Rosstackage
     bool crawled_;
     std::string name_;
     std::string tag_;
+    std::vector<std::string> licenses_;
```

Comment:
Since the new member is private it is not accessible by any external code. What is the purpose here?

Also this change will break ABI which needs to be considered to decide into which branch to merge this.